### PR TITLE
.Net Standard 2.0 support + nuget packaging

### PR DESCRIPTION
**Pull request by fraserd**
Update PelotonEppSdk project to .Net Standard 2.0.

Update PelotonEppSdk project to multi-target .Net Standard 2.0, .Net Framework 4.5.2 and .Net Framework 4.6.1

NuGet package is now built when the project is compiled.

### CODE REVIEW

File: PelotonEppSdkTests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,36 +1,20 @@
...
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
 [assembly: AssemblyTitle("PelotonEppSdkTests")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("PelotonEppSdkTests")]
-[assembly: AssemblyCopyright("Copyright ©  2016")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
```
Comment (cattiwill) [inline PelotonEppSdkTests/Properties/AssemblyInfo.cs]: I would have left this at 2016 ie it is Copyright since 2016

**Review by cattiwill — COMMENTED** [review]
Code changes look good. I did not run the code based on the fact the changes are mainly configuration based, any issues will be picked up pretty quickly by the projects consuming this assembly.

File: PelotonEppSdkTests/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (fraserd) [reply PelotonEppSdkTests/Properties/AssemblyInfo.cs]: Yes, I see. I will revert that specific change. Thanks.

Comment (fraserd) [reply PelotonEppSdkTests/Properties/AssemblyInfo.cs]: Updated in commit https://github.com/pelotontech/Peloton-EPP-.Net-4.5-SDK/pull/19/commits/81a5b8618a1d7b589c85589b3ac4d813fa093e7b

**Comment by fraserd** [comment]
I have tested this code by installing the nuget package into a .Net Standard 2.0 project and installing it into a .Net Framework 4.6.1 project. Both projects compile without warning and are able to access and use all the appropriate SDK classes and methods.

**Pull request merged**